main.py

```diff
@@ -105,6 +127,167 @@ def main():
         elif command == "change-address":
             print(change_address(args, book))
 
+        elif command == "all-notes":
+            print(show_all_notes(notes))
+
+        elif command == "add-note":
```

Comment:
Please add `author` for note.
It should be first step for note creation.
`find-note` command by default should find `note` by `author`